tree-sitter-darklang/grammar.js

```diff
@@ -70,10 +71,28 @@ module.exports = grammar({
       seq(
         field("keyword_type", alias("type", $.keyword)),
         field("name", $.type_identifier),
+        optional(field("type_params", $.type_params)),
         field("symbol_equals", alias("=", $.symbol)),
         field("typ", $.type_decl_def),
       ),
 
+    type_params: $ =>
+      seq(
+        field("symbol_open_angle", alias(token.immediate("<"), $.symbol)),
+        field("params", $.params),
+        field("symbol_close_angle", alias(token.immediate(">"), $.symbol)),
+      ),
+    params: $ =>
```

Comment:
This name is a bit broad - maybe rename to something like `type_params_items` or inline it in the `type_params` rule?